npm/src/language-service/language-service.ts

```diff
@@ -8,6 +8,7 @@ import type {
   IDefinition,
   LanguageService,
 } from "../../lib/node/qsc_wasm.cjs";
+import { PackageType } from "../../lib/web/qsc_wasm.js";
```

Comment:
See prior comment. Add PackageType to the above imports.

Reply:
Oddly enough, making the suggested change here does work, while the previoius location does not. I'm wondering if there is something strange about the settings on my machine...